### PR TITLE
Create pull_request_template.md

### LINKED ISSUE #16: [Doc] Create a pull request template
We're lacking a PR template, we should create one

### PR DESCRIPTION
**Pull request by rheone**
This contribution updates documentation. It adds a pull request template.

Closes #16

### CODE REVIEW

**Review by ahsteele — CHANGES_REQUESTED** [review]
See Gulliver comments

**Comment by rheone** [comment]
Corrected

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Hello, and welcome! Thanks you for your contribution to **Arcus**.
```
Comment (ahsteele) [inline .github/pull_request_template.md]:
```suggestion
Hello, and welcome! Thanks you for your contribution to **Chordly**.
```

**Review by ahsteele — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (rheone) [reply .github/pull_request_template.md]: I suck at this...

**Pull request merged**